dev/tests/integration/testsuite/Magento/Catalog/Model/Product/Attribute/_files/swatch_attribute.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
This fixture is not correct.

1. Rollback is required. Otherwise, an exception `Exception occurred when running the "\/magento2\/magento2ce\/dev\/tests\/integration\/testsuite\/Magento\/Catalog\/Model\/Product\/Attribute\/_files\/swatch_attribute.php" fixture: 
Magento\Framework\Exception\AlreadyExistsException: Attribute with the same code already exists.` is thrown:
```php
<?php
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */

/** @var $attribute \Magento\Catalog\Model\ResourceModel\Eav\Attribute */
$attribute = \Magento\TestFramework\Helper\Bootstrap::getObjectManager()->create(
    \Magento\Catalog\Model\ResourceModel\Eav\Attribute::class
);
$attribute->loadByCode(4, 'swatch_attribute');
if ($attribute->getId()) {
    $attribute->delete();
}
```

2. Attribute is not configured well and tests failed in all data sets with message
```
{"message":"Attribute %1 doesn't work with options","parameters":["swatch_attribute"],"trace":"#0 \/magento2\/magento2ce\/app\/code\/Magento\/Catalog\/Model\/Product\/Attribute\/OptionManagement.php(57): Magento\\Eav\\Model\\Entity\\Attribute\\OptionManagement->add('catalog_product', 'swatch_attribut...', Object(Magento\\Eav\\Model\\Entity\\Attribute\\Option))\n#1 [internal function]: Magento\\Catalog\\Model\\Product\\Attribute\\OptionManagement->add('swatch_attribut...', Object(Magento\\Eav\\Model\\Entity\\Attribute\\Option))\n#2 \/magento2\/magento2ce\/app\/code\/Magento\/Webapi\/Controller\/Rest.php(330): call_user_func_array(Array, Array)\n#3 \/magento2\/magento2ce\/app\/code\/Magento\/Webapi\/Controller\/Rest.php(239): Magento\\Webapi\\Controller\\Rest->processApiRequest()\n#4 \/magento2\/magento2ce\/lib\/internal\/Magento\/Framework\/Interception\/Interceptor.php(58): Magento\\Webapi\\Controller\\Rest->dispatch(Object(Magento\\Framework\\App\\Request\\Http))\n#5 \/magento2\/magento2ce\/lib\/internal\/Magento\/Framework\/Interception\/Interceptor.php(138): Magento\\Webapi\\Controller\\Rest\\Interceptor->___callParent('dispatch', Array)\n#6 \/magento2\/magento2ce\/lib\/internal\/Magento\/Framework\/Interception\/Interceptor.php(153): Magento\\Webapi\\Controller\\Rest\\Interceptor->Magento\\Framework\\Interception\\{closure}(Object(Magento\\Framework\\App\\Request\\Http))\n#7 \/magento2\/magento2ce\/generated\/code\/Magento\/Webapi\/Controller\/Rest\/Interceptor.php(39): Magento\\Webapi\\Controller\\Rest\\Interceptor->___callPlugins('dispatch', Array, Array)\n#8 \/magento2\/magento2ce\/lib\/internal\/Magento\/Framework\/App\/Http.php(135): Magento\\Webapi\\Controller\\Rest\\Interceptor->dispatch(Object(Magento\\Framework\\App\\Request\\Http))\n#9 \/magento2\/magento2ce\/generated\/code\/Magento\/Framework\/App\/Http\/Interceptor.php(24): Magento\\Framework\\App\\Http->launch()\n#10 \/magento2\/magento2ce\/lib\/internal\/Magento\/Framework\/App\/Bootstrap.php(256): Magento\\Framework\\App\\Http\\Interceptor->launch()\n#11 \/magento2\/magento2ce\/index.php(39): Magento\\Framework\\App\\Bootstrap->run(Object(Magento\\Framework\\App\\Http\\Interceptor))\n#12 {main}"}
```

Seems insted of introduction new fixture and resolve these issues fixture `dev/tests/integration/testsuite/Magento/Swatches/_files/swatch_attribute.php` may be reused.